tests/cpp_methods/unittest_cpp_methods.cc

```diff
@@ -92,10 +92,11 @@ TEST (cpp_filter_on_demand, pipeline_01)
   }
 
   if (pipeline) {
-    EXPECT_EQ (setPipelineStateSync (pipeline, GST_STATE_PLAYING, 500000U), 0);
+    EXPECT_EQ (setPipelineStateSync (pipeline, GST_STATE_PLAYING, 500U), 0);
```

Comment:
Let's define a macro for all similar statements and control it centrally.
E.g., ```#define UNITTEST_STATECHANGE_TIMTOUT (500U)```

Reply:
The proposed method seems to be better for managing the tests.
Thank you! Updated.